.gitlab-ci.yml

```diff
@@ -0,0 +1,29 @@
+stages:
+  - test
+  - build
+
+image:                             parity/rust:nightly
+
+variables:
+  CI_SERVER_NAME:                  "GitLab CI"
+  CARGO_HOME:                      "${CI_PROJECT_DIR}/.cargo"
+
+#### stage:                        test
+
+test:rust:stable:
+  stage:                           test
+  script:
+    - time cargo test --all
+  tags:
+    - rust-stable
```

Comment:
What does this tag do? Above you specify `rust:nightly` as image and the tag says stable?

Reply:
This image actually has more than one toolchain:

```
# rustup toolchain list
stable-x86_64-unknown-linux-gnu (default)
beta-x86_64-unknown-linux-gnu
nightly-x86_64-unknown-linux-gnu
```

We only use the tag to assign to a gitlab-runner. But maybe we should just use the tag rust instead. Will change.